RELEASE.md

```diff
@@ -1,3 +1,131 @@
+
+## Breaking Changes
+
+* Premade estimators in the tf.estimator.DNN/Linear/DNNLinearCombined family have been updated to use `tf.keras.optimizers` instead of the `tf.compat.v1.train.Optimizer`s. If you do not pass in an `optimizer=` arg or if you use a string, the premade estimator will use the keras optimizer. This is checkpoint breaking, as the optimizers have separate variables. There will be a converter, but if you want to avoid any change, switch to the v1 version of the estimator:  `tf.compat.v1.estimator.DNN/Linear/DNNLinearCombined*`.
+* `tf.contrib` has been deprecated, and functionality has been either migrated to the core TensorFlow API, to [tensorflow/addons](https://www.github.com/tensorflow/addons), or removed entirely.
+
+
+## Bug Fixes and Other Changes
+
+* `tf.data`:
+  * Adds option for introducing slack in the pipeline to reduce CPU contention, via `options =    `tf.data.Options()`; `options.experimental_slack = True; dataset = dataset.with_options(options)`
+  * Removing the `experimental_numa_aware` option from `tf.data.Options`.
+* `tf.keras`:
+  * Saving a Keras Model using `tf.saved_model.save` now saves the list of variables, trainable variables, regularization losses, and the call function.
+  * `tf.keras.estimator.model_to_estimator` now supports exporting to `tf.train.Checkpoint format`, which allows the saved checkpoints to be compatible with `model.load_weights`.
+  * `tf.keras.model.save_model` and `model.save` now defaults to saving a TensorFlow SavedModel.
+* `tf.estimator`:
+  * Replace `tf.contrib.estimator.add_metrics` with `tf.estimator.add_metrics`
+* `tf.contrib`:
+  * Expose tf.contrib.proto.* ops in tf.io (they will exist in TF2)
+* Other:
+  * Bug fix for `tf.tile` gradient
+  * TF code now resides in `tensorflow_core` and `tensorflow` is just a virtual pip package. No code changes are needed for projects using TensorFlow, the change is transparent
+  * Added gradient for SparseToDense op.
+  * Expose a flag that allows the number of threads to vary across Python benchmarks.
+  * ResourceVariable's gather op supports batch dimensions.
+  * `image.resize` in 2.0 now supports gradients for the new resize kernels.
+  * removed `tf.string_split` from v2 API
+  * Variadic reduce is supported on CPU Variadic reduce is supported on CPU
+  * Added GPU implementation of `tf.linalg.tridiagonal_solve`.
+  * Delete unused lookup table code
+  * Going forward we operate in TF 2.0, this change is part of the effort to slowly converting XYZDataset to DatasetV2 type which is the official version going to be used in TF 2.0 and motivated by some compatibility issue found, _BigtableXYZDataset (of type DatasetV2) does not implement `the _as_variant_tensor()` of DatasetV1, when moving contrib.bigtable to tensorflow_io. Converting into DatasetV2 removes the overheads to maintain V1 while we are moving into TF 2.0.
```

Comment:
Something went wrong with this bullet-- it is hard to interpret as it is written, and we need to back-quote the underscored _Bigtable... to avoid the italicization

Reply:
Done.